packages/vuetify/src/components/VDivider/VDivider.ts

```diff
@@ -1,42 +1,43 @@
-// @ts-nocheck
-/* eslint-disable */
-
 // Styles
 import './VDivider.sass'
 
-// Types
-import { VNode } from 'vue'
+// Utilities
+import { computed, defineComponent, h } from 'vue'
 
-// Mixins
-import Themeable from '../../mixins/themeable'
+// Composables
+import { useTheme } from '@/composables'
 
-export default Themeable.extend({
-  name: 'v-divider',
+export default defineComponent({
+  name: 'VDivider',
 
   props: {
     inset: Boolean,
     vertical: Boolean,
   },
 
-  render (h): VNode {
-    // WAI-ARIA attributes
-    let orientation
-    if (!this.$attrs.role || this.$attrs.role === 'separator') {
-      orientation = this.vertical ? 'vertical' : 'horizontal'
-    }
-    return h('hr', {
-      class: {
-        'v-divider': true,
-        'v-divider--inset': this.inset,
-        'v-divider--vertical': this.vertical,
-        ...this.themeClasses,
-      },
-      attrs: {
-        role: 'separator',
-        'aria-orientation': orientation,
-        ...this.$attrs,
-      },
-      on: this.$listeners,
+  setup (props, { attrs }) {
+    const { themeClasses } = useTheme()
+    const classes = computed(() => {
```

Comment:
Not sure if there's any gain here using a `computed` instead of just putting this class object directly in render function?